plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcPageSink.java

```diff
@@ -238,6 +238,12 @@ public void abort()
                 connection.close();
             }
         }
+        catch (IllegalStateException e) {
+            if (e.getMessage().equals("Connection is already closed")) {
+                return;
+            }
```

Comment:
The logic looks unreliable. The message can be changed in the future, the same message can be thrown in JDBC drivers. How about introducing a dedicated exception class?